wizzard_dialog.py

```diff
@@ -0,0 +1,197 @@
+# encoding: utf-8
+# -----------------------------------------------------------
+# Copyright (C) 2017 Viktor Sklencar
+# -----------------------------------------------------------
+# Licensed under the terms of GNU GPL 2
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+# ---------------------------------------------------------------------
+
+import os
+from PyQt4 import uic
+
+from PyQt4.QtCore import *
+from PyQt4.QtGui import QStandardItemModel, QStandardItem
+
+import trigger_dialog
+from sql_generator import SqlGenerator
+
+this_dir = os.path.dirname(__file__)
+
+WIDGET, BASE = uic.loadUiType(os.path.join(this_dir, 'wizzard_dialog.ui'))
```

Comment:
![image](https://user-images.githubusercontent.com/193367/35384914-0fa9d7b2-01c7-11e8-833a-c28f696610c7.png)

Few things to improve in the dialog:
- dialog title
- align text edit widgets with other widgets
- hide text edit widgets when using "share the same name"
- fix dialog layout so that the tree view takes all free vertical space
- resize tree view columns so that it is possible to see the source table/field names